cardano-testnet/src/Testnet/Components/Query.hs

```diff
@@ -286,3 +289,23 @@ checkDRepsNumber' sbe nodeConfigFile socketPath maxEpoch expectedDRepsNb = do
       H.failure
     Right (_, val) ->
       return val
+
+-- | Obtain minimum deposit amount for DRep registration from node
+getMinDRepDeposit ::
+  (MonadCatch m, MonadIO m, MonadTest m)
+  => H.ExecConfig
+  -> m Integer
+getMinDRepDeposit execConfig = do
+  govState <- H.execCli' execConfig [ "conway", "query", "gov-state"
+                                    , "--volatile-tip"
+                                    ]
+
+  decodedFinalGovFile <- H.nothingFail (decodeStrictText (T.pack govState) :: Maybe Data.Aeson.Value)
```

Comment:
Please use `execCliStdoutToJson` to simplify

Reply:
Addressed [here](https://github.com/IntersectMBO/cardano-node/pull/5748/commits/0fb2ea1fcd90af55bb46667a8a556b7517f72c3d)